src/libraries/System.Net.Http/tests/FunctionalTests/HttpClientTest.cs

```diff
@@ -1226,7 +1226,7 @@ public async Task SendAsync_CorrectVersionSelected_ExternalServer(Version reques
             }
             if (serverVersion == HttpVersion.Version20)
             {
-                remoteServer = useSsl ? RemoteHttp2Server : null;
+                remoteServer = useSsl ? RemoteSecureHttp2Server : RemoteHttp2Server;
```

Comment:
Doesn't this change the test? The test wasn't doing anything in case of `!useSsl`, right?

Reply:
Now it's running the same test over plan HTTP also. Originally `RemoteHttp2Server` had `https://` protocol in it and bad name. Do you prefer to not run it ?

Reply:
> Do you prefer to not run it ?

I'm just wondering why we are changing the test to do something it wasn't doing earlier, which is run for `http://`. If you want to run it over that too, then that should be a different PR.

Reply:
ok, reverted the change